src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -6,12 +6,21 @@
 using BugsnagUnity;
 using UnityEditor.Callbacks;
 using System.Linq;
+using System;
 
 namespace BugsnagUnity.Editor
 {
     public class BugsnagEditor : EditorWindow
     {
 
+        private const string ANDROID_DEPS_XML = "<dependencies><androidPackages><repositories><repository>https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib</repository></repositories><androidPackage spec=\"org.jetbrains.kotlin:kotlin-stdlib:1.5.0\"></androidPackage></androidPackages></dependencies>";
```

Comment:
Ideally we would have some link to the kotlin lib added by the Rakefile so that we end up with the same version and don't risk there being differences depending on EDM support.

At minimum this could be a comment in both places saying to update the other, unless you can see a more sophisticated approach to define the version in one place.

Reply:
Done